OMGWINNING/aa-sdk-staging
# chore: fix up viem versioning and light account integration into daapp

Follow up to:
https://github.com/OMGWINNING/aa-sdk-staging/pull/82
Had some stuff I wanted to land in the vein of testing this out in the Alchemy-DAApp before we merged it.

# Test Plan
1. Ensure All Chains Configured can pull a light account
2. Ensure Alchemy DAApp works correctly
<img width="1728" alt="Screenshot 2023-10-07 at 7 23 10 AM" src="https://github.com/OMGWINNING/aa-sdk-staging/assets/3498866/d2cfdc8a-ee68-4061-b889-99e9437730ba">
